airbyte-integrations/connectors/source-sentry/source_sentry/sentry.yaml

```diff
@@ -0,0 +1,107 @@
+definitions:
+  page_size: 50
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_sentry/schemas/{{ options.name }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      type: JelloExtractor
+      transform: "_"
+  requester:
+    type: HttpRequester
+    name: "{{ options['name'] }}"
+    url_base: "https://{{ config.hostname }}/api/0/"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config.auth_token }}"
+  paginator:
```

Comment:
the test account doesn't have enough data to test the pagination...

Reply:
what about our actual sentry account?

Reply:
Good idea! I was able to test the pagination on the events stream. The same pagination is used for issues and projects. 

`project_details` does not have pagination https://docs.sentry.io/api/projects/retrieve-a-project/